velox/docs/functions/presto/conversion.rst

```diff
@@ -652,6 +652,16 @@ Valid examples
 Cast to Decimal
 ---------------
 
+From boolean type
```

Comment:
Perhaps, add a sentence to clarify that true value is converted to 1 and false to 0.

Reply:
Updated.